ibis/backends/base/sqlglot/compiler.py

```diff
@@ -192,22 +192,181 @@ class SQLGlotCompiler(abc.ABC):
     UNSUPPORTED_OPERATIONS: frozenset[type[ops.Node]] = frozenset()
     """Set of operations the backend doesn't support."""
 
+    SIMPLE_OPS = {
```

Comment:
Can you spell out `ops` for consistency with `UNSUPPORTED_OPERATIONS`?

Reply:
I'd rather do that in a follow up